cmd/krew/cmd/install.go

```diff
@@ -62,6 +62,14 @@ Remarks:
 			var pluginNames = make([]string, len(args))
 			copy(pluginNames, args)
 
+			if *archiveFileOverride != "" && *manifestURL != "" {
```

Comment:
What is the rationale behind forbidding this? Almost nobody will ever try this, I think. But for tests, it could be handy to combine `--manifest-url` and `--archive`.

Reply:
As recommended by @ahmetb here, https://github.com/kubernetes-sigs/krew/pull/350#discussion_r337132131 I added this check. Please advise on how to proceed further.

Reply:
I see, sorry for the confusion. Given that Ahmet had the same idea here https://github.com/kubernetes-sigs/krew/pull/350#discussion_r353975232, I think that we should go for the other option and only treat `--manifest` and `--manifest-url` mutually exclusive, but not `--manifest-url` and `--archive`.

Reply:
Done